sdk/appconfiguration/app-configuration/test/internal/http.spec.ts

```diff
@@ -291,6 +291,38 @@ describe("http request related tests", function() {
       assert.equal(syncTokens.getSyncTokenHeaderValue(), "clearReadOnly=value");
     });
   });
+
+  describe("syncToken", async () => {
+    it("update sync token", async () => {
+      const syncTokens = new SyncTokens();
+      syncTokens.addSyncTokenFromHeaderValue("a=value;sn=0");
+      const client = new AppConfigurationClient(
+        "Endpoint=https://endpoint.azconfig.io;Id=abc;Secret=123",
+        { syncTokens } as InternalAppConfigurationClientOptions
+      );
+      assert.equal(
+        client["_syncTokens"]["_currentSyncTokens"].size,
```

Comment:
You passed in `syncTokens` here so you don't need to delve into the object private fields to get it.

Reply:
client["_syncTokens"] -> syncTokens